src/services/bria/schema.ts

```diff
@@ -2,6 +2,7 @@ import mongoose, { Schema, Document } from "mongoose"
 
 type BriaEventDocument = BriaEvent & Document
 
+// TODO: bring in line with payout augmentation
```

Comment:
Verify payout augmentation is in DB post running tests - if YES we can fix this later.